server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessageFactoryTest.java

```diff
@@ -58,16 +63,31 @@
     private MessageFactory messageFactory;
     private MessagePreviewGenerator messagePreview ;
     private HtmlTextExtractor htmlTextExtractor;
-    
+
+    @Rule
+    public TikaContainer tika = new TikaContainer();
+
     @Before
-    public void setUp() {
-        htmlTextExtractor = new MailboxBasedHtmlTextExtractor(new TikaTextExtractor());
+    public void setUp() throws Exception {
+        tika.start();
+        TikaTextExtractor textExtractor = new TikaTextExtractor(new TikaHttpClientImpl(TikaConfiguration.builder()
+                .host(tika.getIp())
+                .port(tika.getPort())
+                .timeoutInMillis(tika.getTimeoutInMillis())
+                .build()));
+        htmlTextExtractor = new MailboxBasedHtmlTextExtractor(textExtractor);
```

Comment:
ok, you are right, but nonetheless, I just discovered a source of useless waste, I'll create a ticket for that.